ehrql/backends/emis.py

```diff
@@ -64,3 +64,36 @@ class EMISBackend(SQLBackend):
             },
         ),
     )
+
+    ons_deaths = QueryTable(
+        """
+        SELECT *
+        FROM (
+                SELECT Patient_ID AS patient_id,
+                    dod AS date,
+                    Place_of_occurrence AS place,
+                    icd10u AS underlying_cause_of_death,
+                    ICD10001 AS cause_of_death_01,
+                    ICD10002 AS cause_of_death_02,
+                    ICD10003 AS cause_of_death_03,
+                    ICD10004 AS cause_of_death_04,
+                    ICD10005 AS cause_of_death_05,
+                    ICD10006 AS cause_of_death_06,
+                    ICD10007 AS cause_of_death_07,
+                    ICD10008 AS cause_of_death_08,
+                    ICD10009 AS cause_of_death_09,
+                    ICD10010 AS cause_of_death_10,
+                    ICD10011 AS cause_of_death_11,
+                    ICD10012 AS cause_of_death_12,
+                    ICD10013 AS cause_of_death_13,
+                    ICD10014 AS cause_of_death_14,
+                    ICD10015 AS cause_of_death_15,
+                    ROW_NUMBER() OVER (
+                        PARTITION BY ONS_Deaths.Patient_ID
+                        ORDER BY ONS_Deaths.dod ASC
+                    ) AS rownum
+                FROM ONS_Deaths
+            ) x
+        WHERE rownum = 1
```

Comment:
I thought it was too good to be true that the same SQL query executed against both the EMIS _and_ the TPP backend. Then I realized that you haven't tested it against the EMIS backend 🙂 Setting up everything that's needed to do so is beyond the scope of this PR. Consequently, we shouldn't add `EMISBackend.ons_deaths`.

Reply:
I'll delete the QueryTable `ons_deaths` but keep the MappedTable `ons_deaths_raw` in the EMIS backend

Reply:
awrgh, we also need `ons_deaths` in the EMIS backend
```
ehrql.backends.base.ValidationError: <class 'ehrql.backends.emis.EMISBackend'> does not implement table 'ons_deaths' from <module 'ehrql.tables.beta.core' from '/Users/milanwiedemann/Code/opensafely-core/ehrql/ehrql/tables/beta/core.py'>.ons_deaths
```